### DISCUSSION TITLE
GitHub Meetup Official 🥳

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [community/community#29844] [New to GitHub]
Hiya Community! Long-time listener, first time caller in these parts.
Whether you're a beginner or pro, join our meetup group to participate in events with other enthusiasts and developers. 

https://www.meetup.com/pro/github-virtual-meetup/
In Spanish: https://www.meetup.com/githublatam/
In Portuguese: https://www.meetup.com/githubbrasil

**Comment by franciosi** [comment]
TKS for sharing these links!!
